### DISCUSSION TITLE
ColoRIX Image Format

### DISCUSSION THREAD

**Discussion by aaronk6** [ImageMagick/ImageMagick#7211] [Contribute]
As a fun project, I started writing a decoder for the old ColorRIX image format (see http://justsolve.archiveteam.org/wiki/ColoRIX). I have a basic implementation working that can read a RIX3 image with a EGA or VGA palette.

If this has chances getting merged, please let me know. I’m happy to invest more time in testing this thoroughly, and potentially adding support for other features of the format, such as compression. If you aren’t interested in merging this, that’s also cool. Then I have learned how to add a decoder to ImageMagick and will call it a day. Cheers!

**Comment by dlemstra** [comment]
This does sound like an interesting addition so feel free to open up a PR that we can review.

**Reply by aaronk6** [reply]
Thanks for the quick reply. Will do!
